joomla/joomla-cms
# [Media] Remove allowed mime type for flash from new installs

### Summary of Changes

Remove allowed mime type for flash from new installs. Flash is dead but if you really need it you can add it by your self. Per default the core is more secure than 😄 

### Testing Instructions

- review
- install https://github.com/zero-24/joomla-cms/archive/remove_flash.zip
- confirm that flash is not anymore a allowed mime type.

### Expected result

flash is not anymore a allowed mime type.

### Actual result

flash is a allowed mime type.

### Documentation Changes Required

None.
